shared/js/textChatView.jsx

```diff
@@ -442,9 +441,6 @@ loop.shared.views.chat = (function(mozL10n) {
             dispatcher={this.props.dispatcher}
             showPlaceholder={!hasSentMessages}
             textChatEnabled={this.state.textChatEnabled} />
-          <sharedViews.AdsTileView
```

Comment:
@Standard8 FYI, The requirement is now tracked in Aha!:
https://mozilla.aha.io/features/HAO-85#comment_6282609811081516190